### PR TITLE
added terraform scripts for dashboard and cleaned api folder

### LINKED ISSUE #138: terraform the dashboard pipeline
## Task Description

- Dashboard must be hosted on cloud without needing to access it directly.

## User Stories

- Product must be developed by one command to prevent over-complexing deployment.

## Relevant Files [If Available] 

-

### LINKED ISSUE #143: Terraform API ECS
## Task Description

- We need to terraform the ECS that allows our API to run on the cloud

## User Stories

- As a developer, its important we terraform all of our cloud functions, so we can spin it up and down easily and quickly

## Relevant Files [If Available] 

-

### LINKED ISSUE #168: Terraform the dashboard ecs
## Task Description

- Dashboard must be uploaded and hosted on the cloud without needing to deploy it through code.

## User Stories

- Dashboard product must be deployed by one command to prevent over-complexing deployment.

## Relevant Files [If Available] 

-

### PR DESCRIPTION
**Pull request by ErvinRex**
- terraformed the dashboard script
- removed terraform api scripts, keeping only the api terraform scripts in terraform file

closes #138 
closes #143 
closes #168

### CODE REVIEW

File: dashboard/Dockerfile [hunk 1]
```diff
@@ -3,9 +3,11 @@ FROM python:latest
 WORKDIR /app
 
 COPY requirements.txt .
-RUN pip install -r requirements.txt
+RUN pip3 install -r requirements.txt
```
Comment (luciegamby-sigmalabs) [inline dashboard/Dockerfile:6]: I like that you added 'pip3' just to be sure

File: dashboard/Dockerfile [hunk 2]
```diff
@@ -3,9 +3,11 @@ FROM python:latest
...
+
+EXPOSE 8200
 
 COPY streamlit_app.py .
 COPY .streamlit /.streamlit
 
-CMD streamlit run streamlit_app.py
\ No newline at end of file
+CMD streamlit run streamlit_app.py --server.port 8200
```
Comment (luciegamby-sigmalabs) [inline dashboard/Dockerfile:13]: Why port 8200?

File: dashboard/Dockerfile (hunk 1 above)
Comment (ErvinRex) [reply dashboard/Dockerfile:6]: Yeah, weird "fix". Either way it was safer.

File: dashboard/Dockerfile (hunk 2 above)
Comment (ErvinRex) [reply dashboard/Dockerfile:13]: Port 8501 was causing us some issues locally, possibly due to it being used so frequently, so I tweaked the port to run tests locally and just kept it as 8200. No difference in functionality.

**Pull request merged**